cryotempo_eolis_download_plot_gridded_data.ipynb

```diff
@@ -353,14 +213,17 @@
    "outputs": [],
    "source": [
     "fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(20, 10))\n",
-    "dem_list[0].plot(column='elevation', ax=ax, cmap='viridis', vmin=0, vmax=2000, legend=True, legend_kwds={'label': 'Elevation [m]'})\n",
+    "scatter = ax.scatter(gridded_product_data['x'],gridded_product_data['y'], c=gridded_product_data['elevation'], cmap='viridis', vmin=0, vmax=2000, s=20)\n",
```

Comment:
you could use the geopandas plotting function here (and for the thumbnail), converting the geometry column in the geodataframe returned from specklia to the x and y columns with a buffer applied to better show the resolution/spacing of the gridded product. The buffer can be defined as half the grid resolution (which can be taken from the source information)

```suggestion
    import geopandas as gpd
    gridded_product_data.set_geometry(gpd.points_from_xy(gridded_product_data['x'],gridded_product_data['y']).buffer(geospatial_resolution / 2), crs=geospatial_projection, inplace=True)

gridded_product_data.plot(column='elevation', ax=ax, vmin=0, vmax=2000, legend=True, legend_kwds={'label': 'Elevation [m]'})
```

Reply:
Ah perfect, thank you! Have applied this now to both the main plot and the thumbnail :) looking snazzy